### PR TITLE
Release 0.9.0

### PR DESCRIPTION
**Pull request by dbrgn**
Replaces #205.

This also adds a `RELEASING.md` file as internal documentation.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -24,6 +24,25 @@ forget to update the links at the bottom of the changelog as well.-->
 ### Documentation
 
 
+## [v0.9.0] - 2021-12-09
+
+### Additions
+
+- Unify RCC enable / reset ([#196])
```
Comment (dbrgn) [inline CHANGELOG.md:31]: Again: Not sure if this is an addition or breaking / non-breaking change.

Comment (hannobraun) [reply CHANGELOG.md:31]: Strictly speaking it's breaking, due to the addition of trait bounds to at least one `Instance` trait. I can't imagine how that could matter to a user.

File: CHANGELOG.md [hunk 2]
```diff
@@ -141,6 +160,10 @@ _Not yet tracked in this changelog._
 
 <!-- Links to pull requests and issues. Note that you can use "issues"
 in the URL for both issues and pull requests. -->
+[#203]: https://github.com/stm32-rs/stm32l0xx-hal/issues/203
+[#200]: https://github.com/stm32-rs/stm32l0xx-hal/issues/200
+[#197]: https://github.com/stm32-rs/stm32l0xx-hal/issues/197
+[#196]: https://github.com/stm32-rs/stm32l0xx-hal/issues/196
```
Comment (hannobraun) [inline CHANGELOG.md:166]: All those links are broken.

```suggestion
[#203]: https://github.com/stm32-rs/stm32l0xx-hal/pull/203
[#200]: https://github.com/stm32-rs/stm32l0xx-hal/pull/200
[#197]: https://github.com/stm32-rs/stm32l0xx-hal/pull/197
[#196]: https://github.com/stm32-rs/stm32l0xx-hal/pull/196
```

**Review by hannobraun — CHANGES_REQUESTED** [review]
Found some broken links. Looks good otherwise.

**Comment by hannobraun** [comment]
#206 and #208 have been merged in the meantime, in case you want to include more in the release.

File: CHANGELOG.md (hunk 2 above)
Comment (dbrgn) [reply CHANGELOG.md:166]: Ughh, that's new. In the past, `issue` would automatically redirect to `pull`.

Comment (hannobraun) [reply CHANGELOG.md:166]: The links say `issues` though, not `issue`.

Comment (dbrgn) [reply CHANGELOG.md:166]: Sorry, I meant to write `issues` 🙂

I'll update the PR now.

Comment (dbrgn) [reply CHANGELOG.md:166]: Uh, this is weird. In my browser session, https://github.com/stm32-rs/stm32l0xx-hal/issues/203 fails. In an anonymous window (or with curl), it properly redirects to the PR.

Comment (hannobraun) [reply CHANGELOG.md:166]: Oh, I was confused. I thought the correct link to an issue was `/issue/123`, but it's actually `/issues/123`. That's indeed something that used to work.

Comment (hannobraun) [reply CHANGELOG.md:166]: > Uh, this is weird. In my browser session, #203 fails. In an anonymous window (or with curl), it properly redirects to the PR.

Can confirm, same for me! Probably a weird bug that only affects logged in users.

**Comment by dbrgn** [comment]
@hannobraun branch is rebased, CHANGELOG is updated. I also noticed that the publish flag needs valid feature flags (for the pre-checks), I updated `RELEASING.md` accordingly.

**Review by hannobraun — APPROVED** [review]
Thank you, @dbrgn, looks good!

**Comment by dbrgn** [comment]
Thanks, I'll push the release. (Then I can test whether my credentials work now 🙂)

**Comment by dbrgn** [comment]
Release is published and tagged.

**Pull request merged**